<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LocalizationWidget</class>
 <widget class="QWidget" name="LocalizationWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>579</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="layoutWidget2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>781</width>
     <height>561</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>1379</width>
        <height>245</height>
       </size>
      </property>
      <property name="widgetResizable">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>780</width>
         <height>560</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QTableWidget" name="results">
      <property name="columnCount">
       <number>3</number>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>182</number>
      </attribute>
      <row>
       <property name="text">
        <string>Manhattan (Blue)</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>Euclidean (Red)</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>M-Closest (Manhattan) (Green)</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>M-Closest (Euclidean) (Cyan)</string>
       </property>
      </row>
      <column>
       <property name="text">
        <string>X (px)</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Y (px)</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Direction</string>
       </property>
      </column>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>M :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="MValue">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Number of Measurements:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBox">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>20</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="realTime">
        <property name="text">
         <string>Localization OFF</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Device:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="devices"/>
      </item>
      <item>
       <widget class="QPushButton" name="measure">
        <property name="text">
         <string>Measure Position</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
